src/range.js

```diff
@@ -12,11 +12,12 @@ var _isNumber = require('./internal/_isNumber');
  * @sig Number -> Number -> [Number]
  * @param {Number} from The first number in the list.
  * @param {Number} to One more than the last number in the list.
```

Comment:
It would probably be worth updating the description for the `to` param to recognise that it now supports both directions.

Perhaps something like:
```
The exclusive bounds of the sequence. The last number of the returned list will be one less than or more than the `to` parameter for incrementing and decrementing ranges respectively.
```